### DISCUSSION TITLE
shutter control with rotation

### DISCUSSION THREAD

**Discussion by DeFesch** [cgiesche/streamdeck-homeassistant#207] [Help and Support]
Hi,

I want to control my shutters with the dail of the Stream Deck +. I it work with the {{ rotationPercent }}, but on every reboot. the Stream Deck lose the actual state of the shutter and begin with 0%. Is there a possibility to take the actual state of the shutters from HA?

**Comment by jer78** [comment]
I was trying to do a similar thing but with volume controls on media_player and I don't see a way to read the current value to then either add or subtract {{ticks}} to get the new value to send. 

There doesn't seem to be any documentation either and I've tried every combination I can think of. Has anyone got a working example how to use the dial to set a new value based on the rotation of the dial?

**Comment by cgiesche** [comment]
I can confirm, there is currently no way to get/set the "current absolute rotation value" after a reboot. Therefor the "ticks" variable sould be the preferred way to change values. Unfortunatelly, most entities do not offer services that accept "steps" (light.turn_on is one who does).
I will have a look if i can somehow implement this feature.
